src/terrain/index.js

```diff
@@ -0,0 +1,157 @@
+const { getRandomTerrainFunction } = require('./terrainFunctions');
```

Comment:
nitpick: `Function` in the name is redundant. This could be called something like 'helpers' and it would be implied that the helpers are related to 'terrain' since the file is in the 'terrain' directory. Or 'util' or something like that. But again it's not a big deal

Reply:
good call, I dropped function out as I felt helper wasn't really meaningful either. You can correct the new name or not